css/index.css

```diff
@@ -139,47 +170,53 @@ button {
 #messages button {
   margin-left: 1rem;
   max-width: fit-content;
-  background-color: brown;
+  background-color: #00bcd4;
   border: none;
 }
 footer {
-  height: 2rem;
-  background-color: #f9ebd6;
   display: flex;
 
   /* X-axis */
   justify-content: center;
 
   /* Y-axis */
   align-items: center;
+  height: 2rem;
+  background-color: #f9ebd6;
+  
 }
 
 
 .hide {
   display: none;
 }
 
-.nav-bar {
-  width: 100%;
-  display: flex;
-}
 
 #idheader {
   width: 100%;
+} 
+
+.nav-bar {
+  width: 100%;
+  display: grid;
+  grid-template-columns: 2.68fr 1fr;
```

Comment:
Curious about "2.68" ... Sometimes there's math behind particular values that can be explained... and sometimes is just looks right.